src/sources/soundsource.cpp

```diff
@@ -21,6 +21,14 @@ inline QUrl validateLocalFileUrl(QUrl url) {
     return url;
 }
 
+inline QString fileTypeFromSuffix(const QString& suffix) {
+    const QString fileType = suffix.toLower().trimmed();
+    if (fileType.isEmpty()) {
+        return QString{};
+    }
```

Comment:
Why construct a new empty QString if the QString is already empty?

Reply:
To ensure that we end up with *null* String instead of "" for disambiguation. The documentation of QString::trimmed() does not mention what happens in this case, undefined behavior.